### ISSUE TITLE
Atlas walkthrough

### ISSUE THREAD

**Issue by ponderingdemocritus** [BibliothecaForAdventurers/realms-react#150]
The Atlas is a rather complicated UI.

We need to implement a walkthrough like Desiege 

It should at a minimum:
- Explain the interface
- Walkthrough the filters in asset views
- Walkthrough a side panel
- Explain that you can click the assets

**Comment by micksabox** [comment]
Happy to take this. This is enough to start thinking about a walkthrough implementation, UI, persistence and also testing to detect DOM drift

Dropping this here for later
https://javascript.plainenglish.io/testing-local-storage-with-testing-library-580f74e8805b

In terms of UX, I'm thinking an always visible `Guided Tour` button on main menu could re-trigger the flows and even be context aware based on the page being displayed.
<img width="1327" alt="Screen Shot 2022-05-22 at 9 07 37 PM" src="https://user-images.githubusercontent.com/2997810/169725306-9aae2ff6-543a-4923-89c6-33cbc8ac52ac.png">

**Comment by micksabox** [comment]
Waiting for react-joyride to support react 18 https://github.com/gilbarbara/react-joyride/pull/802

**Comment by ponderingdemocritus** [comment]
Assuming we have parked this until support?

**Comment by micksabox** [comment]
Yes, waiting on that PR to go through and next release with React 18 support

**Comment by micksabox** [comment]
Version 2.5.0 was released 9 days ago, this can now proceed. I can get to this once the multi-call approvals are sorted.

**Comment by micksabox** [comment]
Think it's good to wait on implementing this until some more features are settled.
